src/main/java/sparksoniq/spark/iterator/function/ParseJsonFunctionIterator.java

```diff
@@ -22,20 +22,20 @@
 
```

Comment:
@ghislainfourny  in this file and in ParseTextFunctionIterator under the getRDDAux() method I have noticed that the rdd gets cached in _rdd field. Could you remind me in which scenerio this caching helps us ?

Reply:
None. :)

This was from the early alpha version, where we did not pass any dynamic context to getRDD(), making the RDD always the same. But this was incorrect, because the RDD is, actually, dependent on the dynamic context.

So, in the end, unless there is a good reason specific to an iterator, all these caches should go away at some point.

Reply:
Great! I was thinking of refactoring the _rdd field from the class as well as I saw no obvious benefits until I ran across these two caching snippets which made me question their usefulness.

Reply:
We are in Switzerland. You are my guest if you want to clean this up.